### ISSUE TITLE
Nice to have : reorganise photos inside folder

### ISSUE THREAD

**Issue by sebwahl** [thibaud-rohmer/PhotoShow#259]
Would be nice if we could reorder the photos into a folder, and even nicer if it could be done via drag and drop.

**Comment by andreasb242** [comment]
Something like this: http://www.dhtmlgoodies.com/scripts/floating_gallery/floating_gallery.html

**Comment by thibaud-rohmer** [comment]
Yeah... I'm probably not going to implement that. It would imply to add to
much data to the XML files, and they are already a bit too "complete" for
my taste. Of course, feel free to branch and add this functionality :)

**Comment by nodiscc** [comment]
@thibaud-rohmer I suggest you start closing issues and marking them as wontfix if you don't want to be overwhelmed. Currently I use [MinigalNano](https://github.com/sebsauvage/MinigalNano) but may come back to PhotoShow if it gets in a better state (136 open issues :/ ). Thanks for your work on this

**Comment by thibaud-rohmer** [comment]
Yep, I'm planning to do so in a short future. Also, I'll put a "feature
freeze" status quite soon, as I want to start working on other projects.
